shared/ui/main/main.tsx

```diff
@@ -1,7 +1,31 @@
-import { ContainerProps } from '@lidofinance/lido-ui';
 import { FC, PropsWithChildren } from 'react';
-import { MainStyle } from './mainStyles';
+import {
+  MainCard,
+  MainColumn,
+  MainDivider,
+  MainWallet,
+  MainRow,
+  ErrorWallet,
+} from './mainStyles';
 
-export const Main: FC<PropsWithChildren<ContainerProps>> = (props) => {
-  return <MainStyle size="tight" forwardedAs="main" {...props} />;
+type MainSubcomponents = {
+  Wallet: typeof MainWallet;
+  ErrorWallet: typeof ErrorWallet;
+  Card: typeof MainCard;
+  Row: typeof MainRow;
+  Column: typeof MainColumn;
+  Divider: typeof MainDivider;
 };
+
+export const Main: FC<PropsWithChildren> & MainSubcomponents = ({
+  children,
+}) => {
+  return <>{children}</>;
+};
+
+Main.Wallet = MainWallet;
+Main.ErrorWallet = ErrorWallet;
+Main.Card = MainCard;
+Main.Row = MainRow;
+Main.Column = MainColumn;
+Main.Divider = MainDivider;
```

Comment:
```suggestion
Object.assign(Main, {
Wallet: MainWallet,
ErrorWallet: ErrorWallet,
Card: MainCard,
Row: MainRow,
Column: MainColumn,
Divider: MainDivider,
})
```

Reply:
Object.assign works nicely with types - you can do `export const Main = Object.assign(...)` and it will auto-infer types